.github/CODEOWNERS

```diff
@@ -1886,3 +1886,12 @@ spec/uploaders/supporting_evidence_attachment_uploader_spec.rb @department-of-ve
 spec/uploaders/uploader_virus_scan_spec.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/uploaders/validate_pdf_spec.rb @department-of-veterans-affairs/Disability-Experience @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/models/prescription_details.rb @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
```

Comment:
[Line 949](https://github.com/department-of-veterans-affairs/vets-api/blob/master/.github/CODEOWNERS#L949) assigns a codeowner to all files in the `my_health` module:
```CODEOWNERS
modules/my_health @department-of-veterans-affairs/vfs-mhv-secure-messaging @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

The line you're adding here will just apply to `prescriptions_controller.rb`. Is that what you want?
Not saying anything is wrong here 😄, but I just want to double check that this is what you want.

Reply:
ditto with line 1890 below.

Reply:
yes. @rmtolmach do I have to add backend review group, wouldn't the group get notified regardless since they are already over the whole directory?

Reply:
ok 👍 
hmm... 🤔 I see what you mean. But this line overwrites any previous assignments so yes, you do have to add backend-review-group to this line.

I had to check [the docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) 😄 
```
# Order is important; the last matching pattern takes the most
# precedence.
```